cogs5e/initiative/cog.py

```diff
@@ -164,104 +164,53 @@ async def begin(self, ctx, *args):
         await ctx.send(out)
 
     @init.command()
-    async def add(self, ctx, modifier: int, name: str, *args):
-        """Adds a generic combatant to the initiative order.
-        Generic combatants have a 10 in every stat and +0 to every modifier.
+    async def add(self, ctx, name: str, *args):
```

Comment:
Is the removal of the `modifier` arg an intended change? This is technically a breaking change.